assets/_sass/_structure.scss

```diff
@@ -156,6 +156,43 @@
     height: 100%;
     min-width: 375px;
 
+    .article-title {
+      display: flex;
+      align-items: center;
+
+      @include headings {
+        &:hover {
+          cursor: pointer;
+
+          + .permalink .link-icon {
+            visibility: visible;
+          }
+        }
+      }
+    }
+
+    .permalink {
+      padding-left: 8px;
+      cursor: pointer;
+
+      &:hover .link-icon {
+        visibility: visible;
+      }
+
+      .link-icon {
+        display: block;
+        visibility: hidden;
+      }
+    }
+
+    .link-icon {
+      background-image: url("assets/svg/link-icon.svg");
+      fill: #9a9a99;
```

Comment:
Is there no colour var we can use here? Think we need to start try pulling hardcoded hex values out

Reply:
Can introduce a variable, this was from the existing .link-icon that had to move up.